litellm/llms/bedrock/common_utils.py

```diff
@@ -729,7 +729,7 @@ def get_runtime_endpoint(
     api_base: Optional[str],
     aws_bedrock_runtime_endpoint: Optional[str],
     aws_region_name: str,
-) -> str:
+) -> Tuple[str, str]:
```

Comment:
Hey @Manouchehri what is the difference between 'endpoint_url' and 'proxy_endpoint_url'? 

A docstring would help here